buildscripts/pages/gallery.html

```diff
@@ -44,15 +44,15 @@ <h1 class="d-none d-lg-block">udStream Showcase</h1>
     <div class="container-fluid" id="gallery">
 		<div class="row" style="height: 100px;">
 			<!-- Japan -->
-			<div class="col-4 p-1" v-for="project in projects">
+			<div class="col-3 p-1" v-for="project in projects">
 				<div class="card">
 					<a v-bind:href="project.project" target="framebox">
 						<div class="row no-gutters">
-							<div class="col-sm-12 col-md-6 col-xl-4" style="margin: auto; text-align:center;">
+							<div class="col-sm-12 col-md-8 col-xl-4" style="margin: auto; text-align:center;">
 								<img v-bind:src="project.image" class="card-img" v-bind:alt="project.name" style="width: 128px; height: 128px;">
 							</div>
-							<div class="col-md-6 col-xl-8 d-none d-md-block">
-								<div class="card-body p-1 m-1">
+							<div class="col-md-3 col-xl-6 d-none d-md-block">
```

Comment:
`col-md-4` ? With the `8` above to get to 12. `col-xl-6` is either wrong here or above as well (to get to 12)

Reply:
Updated to col-md to 6 and 8. Updated col-xl to 6-6. Responsiveness looks okay.